src/SocketheadCleanArch.API/Utils/AuthHelper.cs

```diff
@@ -0,0 +1,58 @@
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text.Json;
+using Microsoft.IdentityModel.Tokens;
+using SocketheadCleanArch.Domain.Entities;
+
+namespace SocketheadCleanArch.API.Utils;
+
+public static class AuthHelper
+{
+    public static async Task<bool> ValidateAppleTokenSignatureAsync(string idToken)
+    {
+        using HttpClient httpClient = new();
```

Comment:
Never create a HttpClient directly like this in a service class; this is really bad.  This could lead to port exhaustion.  Either create a static client for your class, or better use the IHttpClientFactory.